lib/detect-testing-library-utils.ts

```diff
@@ -32,6 +38,11 @@ export type DetectionHelpers = {
   getCustomModuleImportName: () => string | undefined;
   getIsTestingLibraryImported: () => boolean;
   getIsValidFilename: () => boolean;
+  isGetByQuery: (node: TSESTree.Identifier) => boolean;
+  isQueryByQuery: (node: TSESTree.Identifier) => boolean;
+  isSyncQuery: (node: TSESTree.Identifier) => boolean;
+  isPresenceAssert: (node: TSESTree.MemberExpression) => boolean;
+  isAbsenceAssert: (node: TSESTree.MemberExpression) => boolean;
```

Comment:
I named these functions with prefix `is`. I prefer to use prefix `is`, `has`, `can` for vars, and `getIs`, `getHas` , `getCan` for methods returning booleans. However, `@typescript-eslint/experimental-utils` is using the former for all its utils so I think to follow same standard ours should be named in the same way.

This implies I'll have to rename:
- `getIsTestingLibraryImported` to `isTestingLibraryImported`
- `getIsValidFilename` to `isValidFilename`

I'll do this in a separate PR tho.